docs/tutorials/report-generation/report-tester.collection.json

```diff
@@ -30,10 +30,10 @@
 							"    pm.environment.set(\"holder_did_url\", jsonResponse.didDocument.authentication[0]);",
 							"});",
 							"",
-							"// TODO: fix bug where did documents contain 'services' instead of 'service'.",
+							"// TODO: fix bug where did documents contain 'service' instead of 'service'.",
```

Comment:
I am not arguing to retain that line, but feel it important to caution against global search-and-replace such as that seen here, where the act of fixing broke the statement of the breakage that was being fixed.... --
```suggestion
							"// TODO: fix bug where did documents contain 'services' instead of 'service'.",
```

Reply:
Agree with @TallTed on search/replace, but believe proper fix is to remove the line as you have, in fact, "fixed the bug where did documents contain 'services' instead of 'service'".

Reply:
Haha, @TallTed - you got me! :D 

It was an odd situation where only the incorrect implementations actually worked. I didn't have much time but to quickly project the intention to move in the right direction. 

Glass half full: This is good example of co-development, catching each others mistakes in reviews. Well done!